crux-mir/README.md

```diff
@@ -28,6 +28,15 @@ You will need to install the following software:
 
 ## Setup: `mir-json`
 
+### Download `mir-json` binaries
+
+To download the latest version of `mir-json`, go to the [mir-json action](https://github.com/GaloisInc/mir-json/actions) page. Select the last build from `master` and scroll down the page, until you find and download the desired binary release (`mir-json-macos...` or `mir-json-ubuntu...`). Unzip the downloaded archive and add it to your path.
+
+The binary release comes with precompiled `rlibs`, so don't forget to set `CRUX_RUST_LIBRARY_PATH` variable pointing to the `rlibs` subfolder of the unzipped archive. Adding `export CRUX_RUST_LIBRARY_PATH=...` to your shell configuration is recommended.
+
+
+### Build `mir-json` from source
```

Comment:
I think it would be helpful to include a brief paragraph right after the "Setup: `mir-json`" section header which states that there are two different ways to set up `mir-json`: either by downloading the binaries from GitHub Actions (the "Download `mir-json` binaries" section) or by building it from source (the "Build `mir-json` from source" section). With the way that the subsections are laid out currently, one might be misled into thinking that you have to both download `mir-json` _and_ build it from source, which could be confusing.

Reply:
Good idea, done!